include/envoy/secret/secret_manager.h

```diff
@@ -62,6 +63,13 @@ class SecretManager {
       const envoy::api::v2::auth::CertificateValidationContext& certificate_validation_context)
       PURE;
 
+  /**
+   * @param trusted_ca the protobuf config of the trusted CA certificates.
+   * @return a TrustedCaConfigProviderSharedPtr created from trusted_ca.
+   */
+  virtual TrustedCaConfigProviderSharedPtr
+  createInlineTrustedCaProvider(const envoy::api::v2::core::DataSource& trusted_ca) PURE;
```

Comment:
use TrustedCA type

Reply:
TrustedCA is used for dynamic trusted ca. When we call createInlineTrustedCaProvider(), we pass CertificateValidationContext::trusted_ca field into the method.